container/Makefile

```diff
@@ -101,15 +98,8 @@ ifneq ($(shell id -u), 0)
 	$(error ERROR: You must run the build_local_containers rule as sudo! Please check the README.md file or cd into the container directory and run: `sudo make build_local_containers` followed by `make download_remote_containers`. Current UID: ${curr_uid});
 else
 	$(call log_message,"Building local container $@ ...")
-	$(eval tool := $(basename $@))
+	$(eval tool := $(shell echo $@ | cut -d"-" -f2-4 | sed 's/-[0-9].*//'))
+	$(eval container_url := ${DOCKER_${tool}})
```

Comment:
Hmm ... is `DOCKER_*` a Make variable or a normal env variable? If it is an env variable, it should need double `$` to work: `$${DOCKER_${tool}}`

Reply:
I tested it and it worked! I used the same logic as you did for the fetches from galaxyproject.